docs/user_guide/index.rst

```diff
@@ -8811,6 +8811,219 @@ Environment Code to a target node with Action code, the issue will be
 reported because the two codes are not compatible. The same logic
 applies to the agency ID list.
 
+Manage Business Terms
+---------------------
+Currently, Score provides the business term functionality to end users only. 
+Developers won't see this functionality once login as developer 
+accounts. The end users can view, create, edit or discard business terms through the 
+View/Edit Business Term menu under the BIE menu. The only way to assign 
+business terms to BIEs is through the BIE detail page. At current version, 
+the end users can assign business terms to two BIE types: ASBIE and BBIE. 
+
+Create a Business Term
+~~~~~~~~~~~~~~~~~~~~~~~
+To create a business term:
+
+1. On the top menu of the page, click "BIE".
+   
+2. Choose "View/Edit Business Term" from the drop-down list.
+   
+3. On the returned "Business Term" page, click "New Business Term" located at the top-right
+   of the page. 
+
+4. On the returned "Create Business Term" page, fill out the following fields:
+   1. Business Term (Mandatory) the main name of the business term
```

Comment:
This nested list doesn't get rendered properly. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html